js/_initial_state.js

```diff
@@ -0,0 +1,86 @@
+module.exports = {
+
+	activeStep: 0,
+
+	environment: "production",
+	deployment_type: "Code only",
+	deployment_estimate: "2 min",
+
+	summaryOfChanges: "",
+
+	deployment: {
+		started: false
+	},
+
+	git: {
+		selectedType: "",
+		selectedRef: "",
+		isFetching: false,
+		lastUpdated: 0,
+		list: []
+	},
+
+	approval: {
+
+		bypassed: false,
+		bypassed_time: "",
+
+		rejected: false,
```

Comment:
The thought has indeed struck me, have a `state: "<state>"` instead of the above